d2l-fetch-auth-framed.js

```diff
@@ -1,4 +1,6 @@
 import auth from './es6/d2lfetch-auth-framed.js';
 
+export { auth };
```

Comment:
Exposing `auth` as an export from these global includes, both for testing and for folks to use.